crates/install-wheel-rs/src/script.rs

```diff
@@ -61,10 +64,16 @@ impl Script {
             }
         }
 
+        let function = captures.name("function").unwrap().as_str().to_string();
+        let import_name = function
+            .split_once('.')
+            .map_or(function.as_str(), |(import_name, _)| import_name)
+            .to_string();
```

Comment:
It seems that `import_name` can always be derived by the `function` name and can easily be computed (rather than be stored on `Script`. This is also what `distlib` does. 

https://github.com/pypa/distlib/blob/1c08845b05d022692252ed45cb07e9cb9647caac/distlib/scripts.py#L239-L243

That's why I would remove the `import_name` field and instead add an `import_name` function on `Script` that computes the import name on the fly. This safes us one allocation (good for performance) and avoids storing redudant data.

Reply:
That's fair. I don't think we use the pyo3 interface today. That's why I think it's safe to ignore any concerns regarding it today. 

I don't think we need to store any text ranges. We can return a `&str` and us the same logic that you currently have in the `new` function. I applied the changes to your branch